roles/prereq/tasks/main.yml

```
@@ -82,10 +82,25 @@
       when: groups['server'] | length > 1
       ansible.posix.firewalld:
         port: "2379-2381/tcp"
-        zone: trusted
+        zone: internal
+        state: enabled
+        permanent: true
+        immediate: true
+
+    - name: If firewalld enabled, open inbound ports
+      ansible.posix.firewalld:
+        port: "{{ item }}"
+        zone: internal
         state: enabled
         permanent: true
         immediate: true
+      with_items:
+        - 8472/udp
+        - 10250/tcp
+        - 51820/udp
+        - 51821/udp
+        - 5001/tcp
+        - 6443/tcp
```

```suggestion
    - name: If firewalld enabled, open inbound ports
      ansible.posix.firewalld:
        port: "{{ item }}"
        zone: internal
        state: enabled
        permanent: true
        immediate: true
      with_items:
        - 8472/udp
        - 10250/tcp
        - 51820/udp
        - 51821/udp
        - 5001/tcp
```

You don't need 6443, its the `api_port` opened above.

---

Yes. It is my mistake. Sorry.